frontend/packages/console-dynamic-plugin-sdk/docs/console-extensions.md

```diff
@@ -579,7 +579,7 @@ Adds new Console feature flag driven by the presence of a CRD on the cluster.
 | Name | Value Type | Optional | Description |
 | ---- | ---------- | -------- | ----------- |
 | `flag` | `string` | no | The name of the flag to set once the CRD is detected. |
-| `model` | `ExtensionK8sModel` | no | The model which refers to a `CustomResourceDefinition`. |
+| `model` | `ExtensionK8sResourceIdentifier & { group: string; version: string; kind: string; }` | no | The model which refers to a `CustomResourceDefinition`. |
```

Comment:
The `ExtensionK8sResourceIdentifier & { group: string; version: string; kind: string; }` type is confusing.

```ts
type ExtensionK8sResourceIdentifier = {
    group?: string;
    version?: string;
    kind?: string;
};
```

Therefore, the above type is actually just `{ group: string; version: string; kind: string; }`.

We have a TODO item in [lib-core](https://github.com/openshift/dynamic-plugin-sdk/blob/main/packages/lib-core/src/types/common.ts) SDK package to improve this. cc @florkbr